pkgs/by-name/nm/nmgui/package.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  wrapGAppsHook4,
+  gobject-introspection,
+  gtk4,
+  glib,
+  makeDesktopItem,
+  copyDesktopItems,
+  networkmanager,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "nmgui";
+  version = "1.0.0";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "s-adi-dev";
+    repo = "nmgui";
+    tag = "v${version}";
+    hash = "sha256-HS/n40Ng8S5N14DtEH/upwlxdzwCoOEJA40EMdCcLw4=io";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook4
+    gobject-introspection
+    copyDesktopItems
+  ];
+
+  buildInputs = [
+    gtk4
+    glib
+  ];
+
```

Seems you forgot to list your python dependencies?
```suggestion
  dependencies = with python3Packages; [
    pygobject3
    nmcli
  ];
```

---

Seems I did! When I went back to test, I was using an old nix-store - fixed now!